code-examples/protect-page-login/react/README.md

```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
```

Comment:
Could you add here how to run the Ory Proxy and then the URL to access the React application through the proxy?

Reply:
Ah yes, sorry i wrote this comment with the proxy in mind - not the tunnel. 

But the tunnel is preferred, so this is fine :)